Makefile

```diff
@@ -15,13 +15,13 @@ DIAGRAMS = $(subst src/images,images,$(patsubst %.puml, %.svg, $(DIAGRAMS_SRC)))
 DIAGRAMS_SRC_RAW = $(wildcard src/images/*.svg)
 DIAGRAMS_RAW = $(subst src/images,images,$(DIAGRAMS_SRC_RAW))
 
-SCHEMAS_SRC = $(wildcard schemas/**/*.json)
-SCHEMAS_UTILS_SRC = $(wildcard tools/schemas/**/*.rs)
+SCHEMAS_SRC = $(shell find schemas/ -type f -name '*.json')
+SCHEMAS_UTILS_SRC = $(shell find tools/schemas/ -type f -name '*.rs')
```

Comment:
Q: do we need to replace other `wildcard` calls in this file?

Reply:
`wildcard` seems to have some issues with recursive search, so I had to switch to `find`

will replace the others for consistency and not to fall into the same trap again in case we add more sub-directories